pkg/certificateprovider/azurekeyvault/provider.go

```diff
@@ -95,7 +97,7 @@ func (s *akvCertProvider) GetCertificates(ctx context.Context, attrib map[string
 
 	logger.GetLogger(ctx, logOpt).Debugf("vaultURI %s", keyvaultURI)
 
-	kvClient, err := initializeKvClient(ctx, azureCloudEnv.KeyVaultEndpoint, tenantID, workloadIdentityClientID)
+	kvClientSecrets, err := initializeKvClient(ctx, keyvaultURI, tenantID, workloadIdentityClientID, nil)
```

Comment:
for code readability, I would suggest to make a variable with a meaningful name with nil value to pass into initializeKvClient

Reply:
FYI, looks like you tagged the wrong Josh. @duffney @susanshi

Reply:
sorry about that! updated.

Reply:
Hey @joshuaphelpsms nice to meet you! thanks for the mention. :)